.github/workflows/ci.yml

```diff
@@ -45,12 +45,12 @@ jobs:
   integration-tests:
     name: Integration Tests
     runs-on: ubuntu-20.04
-    container: hashicorp/terraform:1.1.6
+    container: golang:1.18-alpine3.16
     timeout-minutes: 20
     steps:
-      - name: Install Go
-        run: apk add go
```

Comment:
no need to install go - 1.18 is already installed as part of the docker image.
Before it would install go1.17.